src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -94,20 +94,38 @@ const QuickTraceLite = withProjects(
     if (root) {
       const target = generateSingleEventTarget(root, organization, projects, location);
       nodes.push(
-        <EventNode key="root" type="white" icon={null} to={target}>
-          {t('Root')}
-        </EventNode>
+        <Tooltip
+          position="top"
+          title={t('Navigate to the root transaction in this trace.')}
```

Comment:
```suggestion
          title={t('View the root transaction in this trace.')}
```

View is bit shorter than 'navigate to', has the same meaning and is less technical.